.pre-commit-config.yaml

```diff
@@ -43,7 +43,7 @@ repos:
     hooks:
```

Comment:
These are managed via the https://github.com/OCA/oca-addons-repo-template. I'd guess this was from a pre-commit auto update? 

These need to be cleared up before this'll go anywhere, or included as a separate PR.